hw/ip/entropy_src/data/entropy_src_testplan.hjson

```diff
@@ -7,6 +7,7 @@
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
```

Comment:
@senelson7 line 9 you want to remove the `]` and add a comma like line 8. i think that is why the CI fails

Reply:
Done, thanks!